docs/development.md

```diff
@@ -0,0 +1,65 @@
+# Development
+
+## Requirements
+* [Visual Studio 2019](https://visualstudio.microsoft.com/downloads/)
+  * Workloads
+    * .NET desktop development
+    * Desktop development with C++
+  * Individual components
+    * C++ ATL for v142 build tools (x86 & x64)
+* Optional installs:
+  * [Docker Desktop for Windows](https://hub.docker.com/editions/community/docker-ce-desktop-windows/) for building the native Linux binaries.
+  * [WiX Toolset 3.11](https://wixtoolset.org/releases/) and the WiX Toolset Visual Studio 2019 Extension for building the Windows MSI installer.
+
+## Building
+
+The New Relic .NET agent is primarily comprised of two solutions: `FullAgent.sln` and `Profiler.sln`.
+
+### FullAgent.sln
+
+To get started quickly, this is the only solution you need to build. Building this solution creates a number of home directories for each of the target platforms the .NET agent supports:
+
+| Framework | OS | x64 / x86 | Default output location |
+| --------- | -- | --------- | ----------------------- |
+| .NET Framework | Windows | x64 | src/Agent/newrelichome_x64 |
```

Comment:
These home dir locations aren't accurate yet, but they should be once the homebuilder replacement is done.  I dislike having to change it now only to change it again soon, but I also dislike inaccurate documentation. 🤷